pkg/bootstrap/bootstrap_test.go

```diff
@@ -50,12 +80,18 @@ var _ = Describe("Bootstrap", func() {
 					Name:      "bar",
 					Namespace: "foo",
 				},
+				DefaultDir: "default-dir",
 			}))
 		})
 
 		When("the directory is not a git directory", func() {
 			It("returns an error", func() {
-				err := bootstrap.CreateConfig("foo", "bar", temp)
+				err := bootstrap.CreateConfig(bootstrap.Config{
+					GitRepository: profilesv1.GitRepository{
+						Name:      "bar",
+						Namespace: "foo",
+					},
+				}, temp)
```

Comment:
we create this `bootstrap.Config` a lot, worth creating once in a `BeforeEach` and saving the value?

Reply:
Yep